typespec-extension/src/code-model-builder.ts

```diff
@@ -1440,7 +1443,24 @@ export class CodeModelBuilder {
 
         for (const [_, opParameter] of parameters.properties) {
           const serializedName = this.getSerializedName(opParameter);
-          const existParameter = op.parameters.find((it) => it.language.default.serializedName === serializedName);
+          const paramLocation = this.getParameterLocation(opParameter);
+          let existParameter: Parameter | undefined;
+          if (paramLocation === "BodyProperty") {
```

Comment:
use `paramLocation === ParameterLocation.Body`?

Reply:
Here is intended to be `BodyProperty`.

The `opParameter` here is from the operation (when spread, the parameter there be the "body property", not the "body" itself).

Reply:
ok, I see.